src/components/UserName.tsx

```diff
@@ -1,39 +1,57 @@
-import {useState} from "react";
+import {useState, useEffect} from "react";
 import "./userName.scss";
+import {useAppDispatch, useAppSelector} from "../store/hooks";
+import {getUser} from "../store/loginName/userSlice";
+import {AnyAction, Dispatch} from "redux";
 
 type UserNameProps = {
   minSymbol: number;
   maxSymbol: number;
 }
-const UserName = ({minSymbol, maxSymbol}: UserNameProps):JSX.Element => {
-  const userNameRegex = new RegExp('^[A-Z\d]{' + minSymbol + ',' + maxSymbol + '}$', 'i');
+const UserName = ({minSymbol, maxSymbol}: UserNameProps): JSX.Element => {
+  const userNameRegex = new RegExp('^[A-ZА-ЯЁ \d]{' + minSymbol + ',' + maxSymbol + '}$', 'i');
   const [value, setValue] = useState('');
-  const [isError, setIsError] = useState(false);
   const [isBlur, setIsBlur] = useState(false);
+  const [addStyle, setAddStyle] = useState('');
```

Comment:
bad naming for addStyle - looks like function name.

It is better to call [extraStyle, setExtraStyle]